process-engine/skeleton
# Feature/fix config build

## What did you change?

I added `nconf` and `nconfetti` as dependencies to the package.json, and made sure that nconfetti is registered to the correct nconf-instsance

## How can others test the changes?

This seemed to work without the fixes for @Paulomart, but i couldn't build the config before. i always got this error:
```
Running "config" task
Warning: nconf is not defined Use --force to continue.
```

With the fixes on this branch, i can now build without errors, and you should too.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
